packages/vue-vocabulary/src/patterns/VHeader/MenuDropdown.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+  <div class="navbar-item has-dropdown is-hoverable" tabindex="0" aria-haspopup="menu" aria-expanded="false">
+    <a class="navbar-link is-arrowless">
+      {{ $t(label) }}
+      <i class="icon caret-down" />
+    </a>
+    <div class="navbar-dropdown">
+      <slot />
```

Comment:
```suggestion
      <slot default />
```

Not necessary, but I appreciate that it gives extra clarity.